hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -347,6 +347,10 @@
       "ozone.security.enabled";
   public static final boolean OZONE_SECURITY_ENABLED_DEFAULT = false;
 
+  public static final String OZONE_HTTP_SECURITY_ENABLED_KEY =
+      "ozone.security.kerberos.http.enabled";
```

Comment:
This should be better named like ozone.security.http.kerberos.enabled to have a common prefix for http related settings as there will be ozone.security.http.simple.* as well.